Source/JavaScriptCore/inspector/remote/cocoa/RemoteInspectorCocoa.mm

```diff
@@ -346,6 +346,13 @@ static bool canAccessWebInspectorMachPort()
         pushListingsSoon();
 }
 
+void RemoteInspector::connectToWebInspector()
+{
+    dispatch_async(m_xpcQueue, ^{
+        RemoteInspector::singleton().setupXPCConnectionIfNeeded();
```

Comment:
i dont think it's the case, but it might be possible that `WIRServiceAvailabilityCheckNotification` is actually necessary in order for `com.apple.webinspector` to properly be created (or woken up if it already exists)